Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -607,6 +607,12 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND STEAM)
     BUILD_WITH_INSTALL_RPATH true
     INSTALL_RPATH "\$ORIGIN/lib"
   )
+
+  add_custom_command(TARGET dolphin-emu POST_BUILD
```

Comment:
Is there a reason this isn't done all the time?  Is Qt managed by the system or do we have a special build like on Windows?

Reply:
There is no Qt present in the steamrt, so yes, [we provide our own build](https://github.com/OatmealDome/ext-steam-runtime-qt). It's not in a submodule, though. I believe we will just have it on the builder.